packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -0,0 +1,322 @@
+
+> **Note:** The parameters `backchannel_token_delivery_mode`, `backchannel_authentication_request_signing_alg`, and `backchannel_custom_authenticator_id` are only available if the client has `urn:openid:params:grant-type:ciba` defined as one of its allowed `grant_types`. See the [Settings](/docs/reference/api/apps/#settings-10) table on the Apps API Reference page for more information on these new parameters.
+
+**Example request**
+
+```json
+{
+    "name": "oidc_client",
+    "label": "CIBA Client",
+    "signOnMode": "OPENID_CONNECT",
+    "credentials": {
+      "oauthClient": {
+        "token_endpoint_auth_method": "client_secret_post"
+      }
+    },
+    "settings": {
+      "oauthClient": {
+        "client_uri": "http://localhost:8080",
+        "logo_uri": null,
+        "redirect_uris": [
+          "https://example.com/oauth2/callback",
+          "myapp://callback"
+        ],
+        "response_types": [
+          "token",
+          "id_token",
+          "code"
+        ],
+        "grant_types": [
+          "implicit",
+          "authorization_code",
+          "urn:openid:params:grant-type:ciba"
+        ],
+        "application_type": "web",
+        "backchannel_custom_authenticator_id" : "aut5ur07m58W2sQD01d7"
+      }
+    }
+}
+```
+
+> **Note:** You can also use the Apps API to [update an OAuth 2.0 client app to use CIBA](/docs/reference/api/apps/#update-application). See the **Supported parameters** table for the parameters to use.
+
+### Configure an authorization server access policy to use CIBA
+
+You can use either the Okta Org Authorization Server or a custom authorization server with CIBA. In this use case example, we use the `default` custom authorization server.
+
+1. In the Admin Console, go to **Security** > **API**.
+2. On the **Authorization Servers** tab, select the edit icon to the right of the authorization server that you want to use. In this use case example, we are using the `default` custom authorization server.
+3. Select **Access Policies**.
+4. Select the edit icon for the Default Policy Rule.
+5. Select **Client-initiated backchannel authentication (CIBA)** as a grant type and click **Update Rule**.
+
+## Enroll a custom authenticator to use CIBA
+
+Use the Devices SDK and your app to enroll a custom authenticator for the test user. In this use case example, we are using the Okta Authenticator Sample App and the xCode simulator to enroll the authenticator. See the **What you need** section if you haven’t set up the sample app.
+
+1. In the **PushSampleApp** directory of the Okta Devices SDK, run the Okta Authenticator Sample App and then open the mobile app in the simulator.
+2. Click **Sign In**, enter the test user’s credentials, and click **Sign In**.
+3. Complete any additional two-factor authentication required and click **Verify**.
+4. On the Sign in securely using push notifications screen, click **Set up**.
```

Comment:
Might be overkill, but should "Sign in securely using push notifications" be in bold. It's a mouthful without context.